test/test_algorithms.cpp

```diff
@@ -216,6 +275,102 @@ TEST_F(xsimd_reduce, using_custom_binary_function)
     }
 }
 
+TEST(algorithms, reduce_batch)
+{
+    const double nan = std::numeric_limits<double>::quiet_NaN();
```

Comment:
For ARM, vectorization for `double` is available only on 64bits arch. Therefore, this test should be guarded with something like
`#if XSIMD_ARM_INSTR_SET >= XSIMD_ARM8_64_NEON_VERSION || XSIMD_X86_INSTR_SET >= XSIMD_X86_SSE2_VERSION`